README.md

```diff
@@ -16,6 +16,7 @@ Name | Type |  Required | Default | description
 `source` | String | Yes | | this is the location source for the moj-module-webapp, the example implies a github repo containing the moj-module-webapp source
 `product` | String | Yes | | this is the name of the product or project i.e. probate, divorce etc.
 `location` | String | No | UK South | this is the azure region for this service
+`appinsights_location` | String | No | West Europe | the Azure region for App Insights instance, previously limited to 'West Europe' but now avaiable in 'UK South'.  Current default is kept simply to prevent data loss.  New instances should set 'UK South'.
```

Comment:
can you update the usage section too please:
https://github.com/hmcts/cnp-module-webapp#usage